pages/ksphere/kommander/1.2/release-notes/index.md

```diff
@@ -32,19 +32,18 @@ Kommander provides a command center for all your cloud native management needs i
 | **Maximum**        | 1.17.x  |
 | **Default**        | 1.17.3  |
 
-# Kommander 1.2.0 Beta 1 (September 14th, 2020)
+# New Features and Capabilities 
+
+## Support for air-gapped environment 
 
-## Features/Improvements
+## Improved catalog handling
```

Comment:
@juliangieseke we need to expand upon each of these new features and link to the topic that goes into more details. Can you help? Here's the goal:

https://docs.d2iq.com/mesosphere/dcos/2.1/release-notes/2.1.0/#new-features-and-capabilities

Reply:
This link is broken but we did the same thing for Kommander 1.1.0 https://github.com/mesosphere/dcos-docs-site/blob/staging/pages/ksphere/kommander/1.1/release-notes/index.md#new-features-and-capabilities-in-kommander-110

Reply:
uh, not sure I got enough context on them 🙈 

Also, I think we need Product to decide which features we actually want to call out explicitly, not sure who did that for 1.1.0? @samvantran, so you know?

Reply:
GOOD glad we did this for 1.1.0. Not sure how that got lost as a format in 1.2. That link helps me reformat this page again.